platformio.ini

```diff
@@ -30,6 +30,6 @@ build_flags =
 	-DBOARD_HAS_PSRAM
 	-DARDUINO_USB_MODE=1
 	-DARDUINO_USB_CDC_ON_BOOT=1
-	;-D DSE050
+	-D DSE050
 	-D DOPTIGA
```

Comment:
because we have now mostly trust wallets with SE050 i think we could disable the Optega by default